rootfs/scheduler/resources/pod.py

```diff
@@ -719,7 +719,41 @@ def _handle_not_ready_pods(self, namespace, labels):
                     message = "\n".join([x.strip() for x in event['message'].split("\n")])
                     raise KubeException(message)
 
-        return None
+        # Handle cases when no one pods is started (quota check fialed)
+        # If quota is enabled in a namespace for compute resources like cpu and
+        # memory, users must specify requests or limits for those values;
+        # otherwise, the quota system may reject pod creation.
+        # http://kubernetes.io/docs/admin/resourcequota/
+        # or when scale process trigger overuse case for some pods:
+        # For example: used 3/3 pods, but user scale to 4
+        events = self._get_namespace_events(namespace)
+        if len(events) != 0:
+            log = self._get_formatted_messages(events)
+            self.log(namespace, log)
+            raise KubeException(log)
+        else:
+            raise RuntimeError('Unable to start pods, no events in namespace found.')  # noqa
```

Comment:
actually, this code block will always return an error which is not what is intended here. It'll either raise a KubeException when there are events, or it'll raise a RuntimeError if there were no events. Wouldn't we want it to `return None` if there were no bad events?